core/src/main/scala/io/chrisdavenport/monoids/All.scala

```diff
@@ -6,11 +6,11 @@ import cats.implicits._
 
 final case class All(getAll: Boolean) extends AnyVal
 object All {
-  def all[F[_]: Foldable](fa: F[Boolean]): Boolean = fa.foldMap(All(_)).getAll
+  def all[F[_]: Foldable](fa: F[Boolean]): Boolean = fa.forall(identity)
```

Comment:
While an optimization, I  wonder if its a mistake to not use the purpose of the newtype here and in `Any`.